tests/test_consumer.py

```diff
@@ -84,28 +86,25 @@ async def test_offset_type_last(stream: str, consumer: Consumer, producer: Produ
         subscriber_name="test-subscriber",
     )
 
-    messages = [str(i).encode() for i in range(1, 11)]
-    await producer.publish_batch(stream, messages)
-
-    await wait_for(lambda: len(captured) >= 4)
-    assert captured == messages[6:]
+    await asyncio.sleep(0.1)
```

Comment:
we can do this to avoid `sleep(0.1)`
```suggestion
    await wait_for(lambda: captured[-1] == b"4999")
```